coalib/bearlib/languages/__init__.py

```diff
@@ -17,26 +13,31 @@
 from .definitions.html import HTML
 from .definitions.Java import Java
 from .definitions.JavaScript import JavaScript
+from .definitions.Jinja2 import Jinja2
 from .definitions.JSON import JSON
 from .definitions.JSP import JSP
 from .definitions.KornShell import KornShell
 from .definitions.m4 import m4
-from .definitions.Matlab import Matlab
 from .definitions.Markdown import Markdown
+from .definitions.Matlab import Matlab
 from .definitions.ObjectiveC import ObjectiveC
 from .definitions.PHP import PHP
 from .definitions.PLSQL import PLSQL
 from .definitions.PowerShell import PowerShell
 from .definitions.Python import Python
 from .definitions.Ruby import Ruby
 from .definitions.Scala import Scala
+from .definitions.Shell import Shell
 from .definitions.Swift import Swift
 from .definitions.Tcl import Tcl
-from .definitions.Vala import Vala
 from .definitions.TypeScript import TypeScript
-from .definitions.Shell import Shell
-from .definitions.Jinja2 import Jinja2
+from .definitions.Unknown import Unknown
+from .definitions.Vala import Vala
 from .definitions.VisualBasic import VisualBasic
 from .definitions.XML import XML
+
+from .Language import Language
```

Comment:
Don't you think it needs to be below `from.definitions.ZShell import ZShell`?
Also, fix `from.definitions.ZShell import ZShell` to `from .definitions.ZShell import ZShell`.

Reply:
Sorry, only the long list of languages should be re-ordered.

Each of the languages is a `Language` instance.

Therefore loading `Language` should occur before trying to load an instance of `Language`.

Reply:
Hey did't get that can you elaborate @jayvdb

Reply:
The following imports
```
from .Language import Language
from .Language import Languages
```
should not be re-ordered and only the list of instances of languages should.